sections/csgcn/csgcn-chapter.tex

```diff
@@ -252,12 +252,14 @@ \subsection{Convolution layers}\label{subsec:csgcn_is_conv_layer}
 \begin{equation}\label{eq:csgcn_is_gc_layer_user}
      e_{u}^{(l+1)}=\sum_{un\in \mathcal{N}_{u}}\frac{1}{\sqrt{|\mathcal{N}_{un}|} \sqrt{|\mathcal{N}_{u}|}}e_{un}^{(l)}
 \end{equation}
-In \Cref{eq:csgcn_is_gc_layer_item,eq:csgcn_is_gc_layer_user} the GCN layer operation is shown for respectively item and user embeddings, where $\mathcal{N}_{u}$ are the neighboring nodes for user $u$, including both items and user side-information, and $\mathcal{N}_{i}$ are the neighboring nodes for item $i$. 
-A neighbor-node is denoted as $un$ or $in$ for respectively user neighbors and item neighbors, and represents either a user or item side-information node for an $in$ node, and either an item or user side-information node for a $un$ node.
+In \Cref{eq:csgcn_is_gc_layer_item,eq:csgcn_is_gc_layer_user} the GCN layer operation is shown for respectively item and user embeddings, where $\mathcal{N}_{u}$ are the neighbor nodes for user $u$, including both items and user side-information, and $\mathcal{N}_{i}$ are the neighboring nodes for item $i$. 
+A neighbor-node is denoted as $un$ for user neighbors $\mathcal{N}_{u}$, and $in$ item neighbors $\mathcal{N}_{i}$, and represents either a user or item side-information node for an $in$ node, and either an item or user side-information node for a $un$ node.
+$N_{un}$ denotes the set of neighbors for the neighbor $un$ of node $u$, and $N_{in}$ denotes the set of neighbors for the neighbor $in$ of node $i$.
+This means that when neigbor nodes' representations are being propagated, the normalizaton is done with both the number of neighbors for the central node and the number of neighbors for the neighbor node which reprentation is being propagated.
 \\\\
-The convolution layer updates user and item nodes' representations by summing the normalized representations of the neighbor nodes.
-Only the item and user nodes are updated as shown, as these embeddings are the only ones used beyond the convolution layer in the prediction function, further described in \Cref{subsec:csgcn_is_score_prediction}.
-The side-info nodes' representations are also updated in the layer but are not being output from the layer since their representations are integrated into the item and user node representations.
+The convolution layer propagates user and item nodes' representations by summing the normalized representations of the neighbor nodes.
+Only the item and user nodes' propagations are shown, as these embeddings are the only ones used beyond the convolution layer in the prediction function, further described in \Cref{subsec:csgcn_is_score_prediction}.
+The side-info nodes' representations are also being propagated with information about their neighbors in the layer but are not being output from the layer since their representations are integrated into the item and user node representations, and therefore they are not used in the score prediction.
```

Comment:
```suggestion
The side-info nodes' representations are also being propagated with information about their neighbors in the layer, but are not being output from the layer since their representations are integrated into the item and user node representations, and therefore they are not used in the score prediction.
```